src/generic/complex_matrices.cc

```diff
@@ -1078,9 +1075,9 @@ namespace oomph
     }
 
     // Check if the dimensions of this matrix and matrix_in are the same.
-    unsigned long i_nrow = this->nrow();
+    unsigned long this_nrow = this->nrow();
     unsigned long matrix_in_nrow = matrix_in.nrow();
-    if (i_nrow != matrix_in_nrow)
+    if (this_nrow != matrix_in_nrow)
```

Comment:
Ah, apologies if I was unclear. My earlier comment was meant to imply that this may have been previously written by you/Jack as `this_nrow` and after (rightfully) replacing all instances of `this_...` with `i_...`, we encountered the `i_nrow` contraption. I had meant that (what is now) `this_nrow` should be replaced with `n_row`.